PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/resources/META-INF/ruleConfigs.json

```diff
@@ -10,5 +10,15 @@
   "ServiceBusReceiverAsyncClientCheck": {
     "client_name": "ServiceBusReceiverAsyncClient",
     "antipattern_message": "Use of ServiceBusReceiverAsyncClient detected. Use ServiceBusProcessorClient instead."
+  },
+
+  "UseOfBlockOnAsyncClientsCheck": {
+    "async_return_types_to_check": [
+      "reactor.core.publisher.Mono",
+      "reactor.core.publisher.Flux",
+      "com.azure.core.util.paging.PagedFlux",
+      "com.azure.core.util.polling.PollerFlux"
+    ],
+    "antipattern_message": "Use of block() on async clients detected. Switch to synchronous APIs instead."
```

Comment:
There are other types of blocking calls as well like `blockLast`, `blockFirst` etc. So, we should not use `block()` in the message.

Reply:
done:)

Reply:
The message is not updated yet.